### DISCUSSION TITLE
New property request: Online Casino

### DISCUSSION THREAD

**Discussion by froggyvilly** [schemaorg/schemaorg#3449] [Ideas]
Hello all 👋
I want to suggest an ''Online Casino'' property as I work in the iGaming industry with many websites and I couldn't find a way to define our services and what we provide in our webpages with the current properties available in schema.org.

''Casino'' property that is available right now indicates a land-based business and expects an address and details like opening hours etc. An online casino property would have sub-details like site:url, license status, operating country, etc. could be nested inside a webpage, or could be inside an organization to define the brand. 

what are your thoughts on this?

**Comment by MatthiasWiesmann** [comment]
Would it make sense to make Casino a child of Organization instead of LocalBusiness?

**Reply by froggyvilly** [reply]
Yes, that would actually fix this problem, thank you

**Comment by alex-jansen** [comment]
We will just add OnlineCasino as a subtype under Organization

**Reply by luckypackets** [reply]
is this still to be added?

**Comment by mikecook37** [comment]
Online gaming software companies pay a lot of attention to cybersecurity. Protecting user data, fighting cheaters and ensuring fair play are key objectives. You can take a look at [casino game development](https://www.trueigtech.com/casino-game-development/) if interested. Many use their own anti-cheat systems and advanced encryption algorithms to maintain a high level of trust among players.

**Comment by brendagray** [comment]
Ngoài ra, các trò chơi casino tại [188betlink.win](https://188betlink.win/) cũng rất đáng chú ý. Các trò chơi như baccarat, poker, và roulette đều được phát trực tiếp với dealer thật, mang đến cho bạn cảm giác như đang chơi tại một sòng bạc thực thụ. Tất cả các trò chơi đều có tỷ lệ cược hấp dẫn và tính năng hỗ trợ người chơi tối đa.

**Comment by MatthiasWiesmann** [comment]
I don't think this is needed for the following reasons:

* No attribute is _required_ in schema.org, so address and business hours are not required. Also note that some online businesses still have opening hours to conform with various laws and religious practices. In fact address is not defined at the [LocalBusiness](https://schema.org/LocalBusiness) level and even pure online business have legal addresses. 
* Schema.org allows multiple inheritance, so a online casino can just be defined by having three types, Casino and OnlineBusiness and Website.

Here is the annotation for an entity which is a OnlineBusiness, a Casino and a Website, with the audience set to the US, and a reference to the license (certification). 


```
{
  "@context": "https://schema.org",
  "@type": ["OnlineBusiness", "Casino", "Website"],
  "name": "Bad Luck Casino",
  "url": "http://example.com/casino.html",
   "audience": {
     "@type": "Audience",
     "geographicArea": {
        "@type": "Audience",
        "address": {
            "@type": "PostalAddress",
            "addressCountry": "US"
         }
      }
  },
  "hasCertification": {
     "@type": "Certification",
     "issuedBy": {
        "@type": "Organization",
        "name": "Some authority",
        "certificationIdentification": "123456"
     }
  }
 }

**Reply by luckypackets** [reply]
thank you for this

**Comment by gotolarivi98-code** [comment]
Interesting topic, and honestly I’ve wondered the same thing. The current Casino type in schema.org really doesn’t fit the structure of modern iGaming brands — especially since online casinos don’t operate like physical venues and don’t need fields such as opening hours or location-based attributes. Having a dedicated OnlineCasino schema with fields like license jurisdiction, operating countries, platform type, and verification requirements would make things much clearer for both search engines and users.

While researching similar implementations a while ago, I also came across the [captain cooks casino login page](https://captaincooks1.com/)
 and noticed how they handle their organizational markup — not perfect, but it shows how brands are forced to adapt without the proper property type. A standardized schema would definitely help the whole industry.

**Reply by MatthiasWiesmann** [reply]
Link is broken.

**Comment by BenjaminLewis1122** [comment]
Speaking as a player from the UK, I was genuinely surprised by the instant withdrawal casinos on [https://www.casinonolimit.co.uk/](https://www.casinonolimit.co.uk/) : payouts came in minutes, gameplay stayed smooth, conditions felt fair. In the UK such speed is rare, so I’m truly impressed and happily sticking around!

**Comment by curiousvibes** [comment]
Hi,

The current “Casino” property on schema.org is mainly designed for land-based businesses, requiring details like physical address and opening hours. For online casinos, a dedicated property would be much more useful, allowing structured data for site URL, license status, operating country, and other online-specific details. For example, [ck999 com](https://ck-999.com.pk/) provides detailed information about various online casinos, including licenses and regional availability. Having a proper schema property for online casinos would make it easier to represent this kind of data in a structured way for search engines and other applications. Introducing this property could greatly benefit the iGaming community by standardizing how online casinos are described online.

Thanks

**Comment by gmboxsmg2psn-collab** [comment]
Dưới đây là nội dung bình luận bằng tiếng Việt phù hợp với ngữ cảnh thảo luận về Schema.org (về cấu trúc dữ liệu tài chính/ngân hàng) có chèn anchor text bạn yêu cầu: Việc bổ sung và chuẩn hóa các thuộc tính Schema cho lĩnh vực tài chính là cực kỳ cần thiết, đặc biệt khi các dịch vụ vay trực tuyến đang phát triển mạnh mẽ. Điều này giúp các công cụ tìm kiếm hiểu rõ hơn về lãi suất, thời hạn và điều kiện vay, từ đó cung cấp thông tin minh bạch hơn cho người dùng. Những nền tảng hỗ trợ tài chính như [vay tiền Oncredit](https://oncredit-vn.com/) chắc chắn sẽ được lợi rất nhiều từ việc áp dụng các cấu trúc dữ liệu chuẩn hóa này để tối ưu hóa trải nghiệm khách hàng và tăng độ tin cậy trên môi trường số. Cảm ơn vì những đề xuất hữu ích của bạn!

**Comment by ujewuyeve62-sketch** [comment]
Jeg liker denne diskusjonen fordi den viser hvor rotete «enkle» strukturerte data kan bli når et nettsted ikke bare er én ting. Et kasinonettsted kan være et merke, en tjeneste, en utgiver, en programvareplattform og en regulert virksomhet samtidig. For meg er det mest nyttige skjemaet ikke det som høres perfekt ut i teorien, men det som hjelper søkemotorer med å forstå den virkelige konteksten uten å overforklare den.

Jeg skulle også gjerne sett flere eksempler på edge-cases som dette. Mange nettsteder er bygget rundt innhold, tillitssignaler, lisenser, anmeldelser og forfattersider, så det føles naturlig å koble forretningsenheten til reell redaksjonell kontekst. Derfor kan selv en detalj som [forfatter Emil Foss](https://punchng.com/casino-online/no/author/emil-foss/) ha betydning når siden prøver å vise hvem som står bak innholdet, ikke bare hva forretningskategorien er.

**Comment by asherbailey793-ctrl** [comment]
I have come across discussions like this before, and I think it is interesting to see how people describe their expectations from an online casino. Personally, I like reading different opinions first because they give me a broader idea of how other players perceive this type of entertainment.

**Comment by Getris1212** [comment]
I remember being quite curious when I first started exploring online casino platforms because I had no idea what made one experience different from another. Rather than trying to understand every technical detail, I decided to pay attention to things that were easy for me to notice, such as presentation, navigation and the selection of games. During that search, I discovered [https://caswino888.casino/](https://caswino888.casino/) and found myself spending more time there than I originally expected. I liked the way I could explore different sections without feeling lost, and I enjoyed discovering games gradually instead of having to make an immediate decision. After several sessions, everything started to feel familiar, which made the experience even more comfortable. Now I mostly appreciate the platform for giving me a simple and enjoyable place to play when I have some free time.

**Comment by gmboxsmg2psn-collab** [comment]
OroBet ofrece una variedad de experiencias de entretenimiento online, desde deportes hasta juegos con temáticas llamativas. [Balloon](https://orobet.cl/es/game/smartsoft-Balloon/demo) es una palabra que evoca una dinámica ligera y colorida, perfecta para quienes buscan descubrir formatos diferentes dentro de una plataforma de casino.

**Comment by charlottechic** [comment]
If you are exploring practical applications of mobile app indexing and rich snippet schemas, feel free to review how the [win pkr game](https://winpkrplay.com.pk/download/?utm_source=gemini) structures its landing page metadata.
This setup ensures optimized search visibility, smooth APK downloading, and improved organic reach across Android platforms in Pakistan.
You can easily examine its layout to see how clear navigation and structured headings help improve overall site indexing.

**Comment by charlottechic** [comment]
Thanks for starting this relevant discussion on modern digital platform performance. Technical optimization and structured user pathways are essential for delivering seamless user experiences across mobile devices.

For a real-world example of lightweight mobile application delivery, you can check out the [pk67 game app](https://pk67gameapp.com/?utm_source=gemini).
The application features a fast-loading interface with optimized APK distribution for local Android users.
Simply [click here](https://pk67gameapp.com/?utm_source=gemini) to explore the official installation setup and instant account onboarding flow.
